js/src/modules/_Menu.es6.js

```diff
@@ -175,7 +183,8 @@ export class PopupMenu extends Menu {
   }
 
   setFocusToItem(newItem) {
-    newItem.focus();
+    //eslint-disable-next-line
```

Comment:
Can we remove this? What linting problem are you hitting with the next line?

Reply:
I had a debugger statement after the line. Forgot to remove the comment